riscv/processor.cc

```diff
@@ -454,6 +454,7 @@ void state_t::reset(processor_t* const proc, reg_t max_isa)
                               (proc->extension_enabled(EXT_SVPBMT) ? MENVCFG_PBMTE : 0) |
                               (proc->extension_enabled(EXT_SSTC) ? MENVCFG_STCE : 0) |
                               (proc->extension_enabled(EXT_ZICFILP) ? MENVCFG_LPE : 0);
+                              (proc->extension_enabled(EXT_ZICFISS) ? MENVCFG_SSE : 0);
```

Comment:
Danger!  The previous statement ends in a semicolon, so this statement has no effect.  Change the previous statement to end in a `|`.